tidoop-mr-lib-api/src/server.js

```diff
@@ -23,36 +23,134 @@
  * Author: frb
  */
 
-// requires
+// module dependencies
 var Hapi = require('hapi');
-var config = require('../conf/tidoop-mr-lib-api.json');
-var pjson = require('../package.json');
+var run = require('./cmd_runner.js').run;
+var version = require('../package.json').version;
+var port = require('../conf/tidoop-mr-lib-api.json').port;
+var tidoopMRLibPath = require('../conf/tidoop-mr-lib-api.json').tidoopMRLibPath;
+var tidoopMysql = require('./mysql_driver.js');
+var serverUtils = require('./server_utils.js');
 
 // create a server with a host and port
 var server = new Hapi.Server();
 
 server.connection({ 
     host: 'localhost',
-    port: config.port
+    port: port
 });
 
-// add the route
+// create a connection to the MySQL server
+tidoopMysql.connect();
+
+// add routes
 server.route({
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-tidoop/commit/6a0bdc08d70a95d05f2982f185aad4e716939f5f